geneva.spec

```diff
@@ -4,12 +4,20 @@ Source: git+https://stash.desy.de/scm/geneva/geneva-public.git?obj=master/%{real
 
 BuildRequires: cmake gmake
 
-Requires: python py2-setuptools py2-numpy gsl boost lhapdf hepmc openloops pythia8
+Requires: python py2-setuptools py2-numpy gsl boost lhapdf hepmc pythia8
+%ifnarch ppc64le
```

Comment:
not sure whether this will allow us to have a fully working version, but I understand that oterwise the build cannot proceed

Reply:
We do not build openloops for non-amd64 archs. We will change the above line to be `%ifarch x86_64` to make it consistent with sherpa, herwigpp and cmssw-tool-conf.

https://github.com/cms-sw/cmsdist/blob/8a4f8beb18943981d26b0b72b295e923a9b6223c/sherpa.spec#L13
https://github.com/cms-sw/cmsdist/blob/ab6cf5ba4201f09658ff066c7d840f5a5e065023/herwigpp.spec#L16
https://github.com/cms-sw/cmsdist/blob/95e43bbbf2ca2530091b8e44b19b4d4fa8aa1539/cmssw-tool-conf.spec#L178